spec/requests/api/users_spec.rb

```diff
@@ -110,17 +110,22 @@
     end
 
     it 'should return 400 error if name not given' do
-      post api('/users', admin), email: 'test@example.com', password: 'pass1234'
+      post api('/users', admin), attributes_for(:user).delete(:name)
```

Comment:
Tests are still wrong, because `delete` returns the value of the deleted key, not a modified `Hash`. 
You should use the `except` method.